client/state/sites/products/conflicts.ts

```diff
@@ -1,31 +1,60 @@
 import {
 	planHasFeature,
+	planHasAtLeastOneFeature,
 	planHasSuperiorFeature,
 	FEATURE_JETPACK_BACKUP_REALTIME,
 	FEATURE_JETPACK_BACKUP_DAILY,
 	JETPACK_PLANS,
-	PLAN_JETPACK_FREE,
-	PLAN_JETPACK_BUSINESS,
-	PLAN_JETPACK_BUSINESS_MONTHLY,
-	PLAN_JETPACK_PERSONAL,
-	PLAN_JETPACK_PERSONAL_MONTHLY,
-	PLAN_JETPACK_PREMIUM,
-	PLAN_JETPACK_PREMIUM_MONTHLY,
-	PLAN_JETPACK_SECURITY_DAILY,
-	PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
-	PLAN_JETPACK_SECURITY_REALTIME,
-	PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
-	PLAN_JETPACK_COMPLETE,
-	PLAN_JETPACK_COMPLETE_MONTHLY,
 	PRODUCT_JETPACK_BACKUP_DAILY,
 	PRODUCT_JETPACK_BACKUP_REALTIME,
 	PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY,
 	PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY,
+	FEATURE_JETPACK_BACKUP_DAILY_MONTHLY,
+	FEATURE_JETPACK_BACKUP_REALTIME_MONTHLY,
+	FEATURE_JETPACK_BACKUP_T1_MONTHLY,
+	FEATURE_JETPACK_BACKUP_T1_YEARLY,
+	FEATURE_JETPACK_BACKUP_T2_MONTHLY,
+	FEATURE_JETPACK_BACKUP_T2_YEARLY,
 } from '@automattic/calypso-products';
 import { createSelector } from '@automattic/state-utils';
-import { hasSiteProduct, getSitePlanSlug } from 'calypso/state/sites/selectors';
+import getRewindCapabilities from 'calypso/state/selectors/get-rewind-capabilities';
+import { getSitePlanSlug } from 'calypso/state/sites/selectors';
 import type { AppState } from 'calypso/types';
 
+/**
+ * Check is a Jetpack plan is including a daily backup feature.
+ *
+ * @param {string} planSlug The plan slug.
+ * @returns {boolean} True if the plan includes a daily backup feature.
+ */
+export const planHasDailyBackup = ( planSlug: string ): boolean => {
+	const DAILY_BACKUP_FEATURES = [
+		FEATURE_JETPACK_BACKUP_DAILY_MONTHLY,
+		FEATURE_JETPACK_BACKUP_DAILY,
+	];
+
+	return planHasAtLeastOneFeature( planSlug, DAILY_BACKUP_FEATURES );
+};
+
+/**
+ * Check is a Jetpack plan is including a real-time backup feature.
+ *
+ * @param {string} planSlug The plan slug.
+ * @returns {boolean} True if the plan includes a real-time backup feature.
+ */
+export const planHasRealTimeBackup = ( planSlug: string ): boolean => {
+	const REALTIME_BACKUP_FEATURES = [
+		FEATURE_JETPACK_BACKUP_REALTIME_MONTHLY,
+		FEATURE_JETPACK_BACKUP_REALTIME,
+		FEATURE_JETPACK_BACKUP_T1_MONTHLY,
+		FEATURE_JETPACK_BACKUP_T1_YEARLY,
+		FEATURE_JETPACK_BACKUP_T2_MONTHLY,
+		FEATURE_JETPACK_BACKUP_T2_YEARLY,
+	];
```

Comment:
This can also be moved out

Reply:
Moved out in [99dda55](https://github.com/Automattic/wp-calypso/pull/61525/commits/99dda556776192958851ab244df2474310ad87c5) 👍